docs/tutorial/05-loops-conditions-existing.md

```diff
@@ -70,7 +70,7 @@ resource foo 'my.provider/type@2021-03-01' = [for <ITERATOR_NAME> in <ARRAY> = {
 
 `ITERATOR_NAME` is a new symbol that is only available inside the body of the resource declaration. It can be any name you would like and represents the current item in the array.
 
-Let's remove our condition and replace it with a loop based on an input of container names:
+Let's remove our condition and replace it with a loop based on an input of container names. There's also a variant which gives us access to the current item's index in the array:
```

Comment:
Good addition - can we add a link here to the index-based loops part of the loops.md spec?